calico_rkt/calico_rkt.py

```diff
@@ -146,15 +152,20 @@ def execute(self):
             _log.debug("Starting plugin execution.")
             self._execute()
         except SystemExit, e:
-            # SystemExit indicates an error that was handled earlier
-            # in the stack.  Just set the return code.
-            rc = e.code 
+            # Defer printing the error message and setting the rc in the
```

Comment:
This comment should indicate that the error was handled elsewhere in the code, so we don't need to log or print anything here.

Might be worth a debug log anyway that says "Handling SystemExit" or something so we can better trace execution.